public/401.html

```diff
@@ -1 +1,87 @@
+aE04th1ehpFKUNWBCm2CBKexuE2/WeaXdgofATpTqoMVwl/65KmBGnnPzCf9s29N+jIe0loupCcFldH8qD8oBVVs84g5VhXWE0IpQMIVifq5Mr1hUWj/3W5v6vO4hb4QF3zINTWJSvXWxnd7ATXEIUfV4lSgXRSKJ7QajZ661cy4XNRn+W/uylyYdzRX07UVVfCKQjjcH2zK0Su3KjjV1XmSub1WD10cuGPuEAVhBWBSu2j2nQJYRVtVSU4ldIQp1rV3C1O7CsTSR75EWc9ehCXiwcJuESWvYEWFUrnq7kChaTWdfu4GJ1h/rFNf2/JgbDIwtiWi5WBKGkAaujVT9740H3iDi/pReD7kH+jnJ4mfmLFZp1wkGx9RPjqdVPnUg9UNDFGmKpurCTD5JMm21dLdaW7jeLrT1vdgsrQORgrZwrrtbbt0DaXXqZj++VMEscxbPeurzryE0Xdf1WWBOHeRtWl+2VQcO60jPe1AzC0kKulktY4b4bZ4IogeU3TctBaCmUrb7vRV3CSv0zCflZ1w+8QyyNjriBFXTKgFWKV5LA8qMQ8HQU1LegZTy9D/H0XShksNfllQErPafeugKsbh58byOwQr7VNK+iOnVypjtgrXngLuJ11wmHQXiU8Hxj7+w3GYi/UN868bnq/lYeSaJZIQpqO6ggI/7mgfeK3kj8yY9fPvQRF7CqGWS/4HvYm4e4p+LyvkUSdL4TJ20nChcefd/RRnZQ8XkAAA86vZmWd0fSf7R96H3M01qoejvrmK6gYlghXgVY9UfjjzUDVg7uWfjH2+VYLjpwX9FOBBaeTuh1jeb81xhji1CbAV8OLceWG6DFjPjzQt+s6rDKKQ+0YzXw+v53iqXxxP0u8qyEfFAd9/Ahe5EE14gorU4TWB0ALKvve7Tsp2oBWo7KdywhI34oFvpr4qp6Cxo9XzRjV6q8eTO4boDq8p7rjqE28J4tg/e5hNVBj1YE8VkQH0W7YwTxbW9B11aTt0NjWEgkvUSUMoSRAY/HuFetKVEOtEq42BDj2/tnbzwwk/t0p+ZqIf8KeVgVnwYKc66WRleZpUEDkf7d1y3r+oSDrgvKYGVjHmryXuqXf54VbdSrncCqf/ET0rICqLIeX3jX0EKC6S9Opv88W9Sv7aQLhE4NgFWlPCw9nStltCu4ZFZCaHdE+/69Wwf/2IVVpcSysjkPtbLhdW4hgdXhcg0t6CuvTu84lS78SSecKHQWRa+sSgF3s3VMsvFVQawCXm1YVUPhpcnRrvBnP7B54JFWwcoBsDpeBFZzBYsO3SNcre786PD8lpcnsp9qZ2urFqyQIFpMZkrtY9xeAC1uWFXXiU3dV5lW1VVL4vffMNp1xCWs8CGPqohZcR4SWH6VqzytTrG20LAP7mAlwbLS0+63nEe6wraem0VvuG9iNBH+nw1YVZDS1AXOQwKrUahgFDyKE+C10XSt2+0LPHp4fvPYdO7uuVzxrnayrrAbTsUJjbhVKrugR7tdIai+tec6MWL8xEYR92wZ/KsGQAVNG7BSupdg2TxE54W4vEcQX81xOhJY9dw2JJMi5oSVGER9vSpcRT5NQ1sf379n5l0nU/n/nNfFxqCf+GqtZMy2Mamc47gVipYRUF+b2CISIfHEyt7od7GrTQOwwmQ5W62nlQJgWb2t0D0UqTdH5SC0CKyqls+QMa6UIEECIHqbeFm8ivdBT26tkRdpB3BV2mgC+VZ6KuOoiygC6hu7tqsElZDW9nGvCpnlPIzJcMFlEl6A1cuilL5AEVgVgYVM4mulWT5pDGxo5/VOnLDmlJRaBBlcSGFY0L8dQXbDDbSbb1Xu+ikEFaQsuF4HWBEZKrhUwgvBPC9zBAmsoJ8PUWp3vEm6hePG2CdKeVrNgGVDca0LwJUu3JMv6puDdhGsYmc7yaFY9cNONxu7rhQ94X7VoIImFW8aUQtYVn7ggIQXHpRt2+6YwFIHDlhXzUoq9cRFtIQ9EY/M5+9K5fW3B+HkW1t7mZZVqvpzAit+a+NbzZ+a0CcGYpF/uGE0/tBlQ/GzikCF3InxZvWyKpuHHZ8cSmAFE5qrhMKibMNF1L42Nn33RLb4G362urBKeGbmaCl9oezehCU1ElspVsQ2GONis8d6LCSeWZoIP7JjY/+PFEHKkvJVQAfAogiswEqptWVp50Rm6Mnx1F0zOf3X/QQvBN33TT4vZufPmhtJICY1El1pggpxKQiQ6o+Gn/zgJQN/HzamuWJQNdWqIrAIrHa1tpD+0OcJJEx4pf9jMl+8OlPQr9Y9ep96QmY7Uhq6ZqNiILTkHKDg7sXD2s7eSPhZjyAlpNs10wyryphvmgw3YBUwZ7xnnsAisLwUbrQu+RNP4myT3rdLgivm5Zs8fGDuTSdThauTheLluYK+oaCLVXrJNdW8nsshTUzNnRqfTETDu3rD4oUNA/HnbhxNHPIAUOVCy5aTBjg8z3WSsIpLdx8PhrPSostxHhJYXsEKLhqWnTGJ0YbkjSZCC0ITtyHR5O3IAbL5fHHJZKZ4dVGInlxRX1Eo6ud2iDD+bWlRN1dWLzxpmjEpy3KJomFtf1jTZgzTcWYoHnqpJxI6+76Le19otlFnjDOitArXTGsZiaFXi1ITSbz/803+DARWBwnWDTZNvVzefEh7eLFFN1xLwNUGagWoyoGF63YNgUVgNcvCGpUWVpe0rpqVp0VwBRdU5YJljr0E+i2XULD0hsDyUFGxMIaFXC0/nESACy5ZhJdogZClPuEjK0aT1ygiQZXnJSKwmnXj+fHk9UpwdXfwtdEloM7QeiGwqGDICvD2dZDVBWvK8Q7cHjzIrIcZJxeBFQgBEDF5w8JlbHUtmNUypx3hlZbW1JQPzjNA1SMHLLtZWngElt8FOKDjAwqZEeg9LEpdH4o++nyDclLFAnh+ccMidpj0CaTKhYA6dmNaKyH6inRLC5wW6iwBSj0Q0CoGyYF5eePOitauJJYLT/zTckC9El5dPgWYZaVae/Glf
```

Comment:
This text isn't right, the app is available... but access is denied. So this should read "401 - Access Denied"